lua/weapons/bobs_gun_base.lua

```diff
@@ -904,11 +913,16 @@ function SWEP:GetViewModelPosition( pos, ang )
     pos = pos + Offset.y * Forward * mul
     pos = pos + Offset.z * Up * mul
 
-    if self.RecoilAmount > 0 then
+    if self.RecoilAmount > 0 and self:GetIronsightsActive() then
         local forward = ang:Forward()
         local recoilOffset = forward * -self.RecoilAmount
         pos = pos + recoilOffset
-        self.RecoilAmount = Lerp( math.ease.OutCubic( FrameTime() * self.RecoilRecoverySpeed ), self.RecoilAmount, 0 )
+
+        local easer = math.ease.OutCubic(self.RecoilRecoverySpeed * (CurTime() - prevThinkBlowback))
```

Comment:
Make sure to follow cfc style guidelines https://github.com/CFC-Servers/cfc_glua_style_guidelines